yaml/places.yaml

```diff
@@ -176,6 +176,18 @@ filters:
       kind: region
       kind_detail: province
     table: osm
+  - filter: {name: true, place: unrecognized}
```

Comment:
Considering consolidating this into the existing section above

> `- filter: {name: true, place: country}`

As

> `- filter: {name: true, place: [country,unrecognized]}`

With 

>       kind: unrecognized

Changing to 

>      kind: {col: place}

Reply:
Around line 141

Reply:
nice. even simpler